files/slc_clean

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+show_help() {
+    echo 'Options:
+    -s, --svc               Service ID (default: 1)
+    -c, --count             Version kept (default: 2)
+    -h, --help              Show this help
+'
+    echo 'List Services:'
+    if hash slc 2>/dev/null; then
+        slc ctl ls
+    fi
+}
+
+
+get_options() {
+    OPTION=$(getopt --options s:c:h \
+             --long svc:,count:,help \
+             -- "$@")
+
+    # exit if the options have not properly been gathered
+    if [ $? != 0 ] ; then echo "Terminating..." >&2 ; exit 1 ; fi
+
+    # Note the quotes around '$OPTION': they are essential!
+    eval set -- "$OPTION"
+
+  while true ; do
+      case "$1" in
+        -s|--svc) SVC="$2" ; shift 2 ;;
+        -c|--count) COUNT="$2" ; shift 2 ;;
+        -h|--help) show_help ;exit 1 ;;
+        --) shift; break;;
+        *) echo "Internal error!" ; exit 1 ;;
+      esac
+    done
+}
+get_options "$@"
+
+# Set the defaults if no data provided on the CLI
+if [ -z "$SVC" ]; then
+    SVC=1
+fi
+
+if [ -z "$COUNT" ]; then
+    COUNT=2
+fi
+
+
+WORKDIR=/var/lib/strong-pm/svc/$SVC/work
+cd $WORKDIR
```

Comment:
Check that $WORKDIR exist before going in - if not, print error message and exit.